From: Diego Escalante Urrelo <diegoe@gnome.org>
Date: Thu, 13 Feb 2025 05:40:26 -0500
Subject: wl: Use kernel keyword for fallthrough

The comment convention leads to false positives with the macro expansion.
---
 amd64/src/wl/sys/wl_cfg80211_hybrid.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/amd64/src/wl/sys/wl_cfg80211_hybrid.c b/amd64/src/wl/sys/wl_cfg80211_hybrid.c
index e23ef83..e5e6eb9 100644
--- a/amd64/src/wl/sys/wl_cfg80211_hybrid.c
+++ b/amd64/src/wl/sys/wl_cfg80211_hybrid.c
@@ -811,7 +811,7 @@ wl_set_auth_type(struct net_device *dev, struct cfg80211_connect_params *sme)
 		break;
 	case NL80211_AUTHTYPE_NETWORK_EAP:
 		WL_DBG(("network eap\n"));
-        /* falls through */
+		fallthrough;
 	default:
 		val = 2;
 		WL_ERR(("invalid auth type (%d)\n", sme->auth_type));
